CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## 0.20
 
+- add `{QCheck,QCheck2}.Gen.{bytes_size,bytes,bytes_of,bytes_printable,bytes_small}`
+- add `QCheck.{Print,Shrink,Observable}.bytes`
+- add `QCheck2.{Print,Shrink}.bytes`
+- add `QCheck.{bytes_gen_of_size,bytes_gen,bytes,bytes_small,bytes_of_size,bytes_printable}`
+- add `{QCheck,QCheck2}.Gen.{bytes_size,bytes,bytes_of,bytes_printable,bytes_small,string_small}`
+- add `QCheck.{Print,Shrink,Observable}.bytes`
+- add `QCheck2.{Print,Shrink}.bytes`
+- add `QCheck.{bytes_gen_of_size,bytes_gen,bytes,bytes_small,bytes_of_size,bytes_printable}`
+- add `QCheck.{string_small,string_printable,string_printable_of_size,string_small_printable,string_numeral,string_numeral_of_size}`
+- `QCheck2.{small_string}` character generator argument is no more optional
```

Comment:
This still contains redundancy between the individual lines.
I propose reducing it to:
```
- add `{QCheck,QCheck2}.Gen.{bytes_size,bytes,bytes_of,bytes_printable,bytes_small}`
- add `QCheck2.Gen.string_small`
- add `QCheck.{Print,Shrink,Observable}.bytes`
- add `QCheck2.{Print,Shrink}.bytes`
- add `QCheck.{bytes_gen_of_size,bytes_gen,bytes,bytes_small,bytes_of_size,bytes_printable}`
- add `QCheck.{string_small,string_printable,string_printable_of_size,string_small_printable,string_numeral,string_numeral_of_size}`
- `QCheck2.small_string` character generator argument is no more optional
```